images/tester/tests/test_dag_consistency.py

```diff
@@ -14,4 +15,7 @@ def test_all_dags_in_dag_bag(dag_bag, dag_file):
         for dag in dag_bag.dags.values()
     }
 
+    # Execute the file. If it fails to execute, we should see a traceback why.
+    subprocess.call(["python", DAG_PATH + dag_file])
```

Comment:
This may not be a correct invocation, as the PYTHONPATH needs to be set correctly such that the airflow dags directory is on the python path.

Reply:
relative to the location at which the tests are executed, i feel like this has to be correct, if we were finding them in the first place?

Reply:
See: https://github.com/apache/airflow/blob/1.10.5/airflow/models/dagbag.py#L354